src/Microsoft.ML.Transforms/EntryPoints/TextAnalytics.cs

```diff
@@ -158,5 +158,18 @@ public static CommonOutputs.TransformOutput WordEmbeddings(IHostEnvironment env,
                 OutputData = view
             };
         }
+
+        internal sealed class ReadonlyMemoryComparer : IEqualityComparer<ReadOnlyMemory<char>>
```

Comment:
>ReadonlyMemoryComparer [](start = 30, length = 22)

just wondering if this is the best place for this. Is there any utils classes to host it?